admin_guide/multiproject_quota.adoc

```diff
@@ -0,0 +1,100 @@
+[[admin-guide-muliproject-quota]]
+= Multi-Project Quotas
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
+to be shared across multiple projects.  Resources used in each selected project
+will be aggregated and that aggregate will be used to limit resources across
+all the selected projects.
+
```

Comment:
We need to add an anchor just abovethe heading:
[[multi-project-quotas-selecting-projects]]